src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.Client.fs

```diff
@@ -127,17 +127,21 @@ module Client =
         |> Seq.choose (analyzerFromMember<'TAnalyzerAttribute, 'TContext> path)
         |> Seq.toList
 
-type Client<'TAttribute, 'TContext when 'TAttribute :> AnalyzerAttribute and 'TContext :> Context>
-    (logger: ILogger, excludedAnalyzers: string Set)
-    =
+type ExcludeInclude =
+    | Exclude of string Set
+    | Include of string Set
+
+type Client<'TAttribute, 'TContext when 'TAttribute :> AnalyzerAttribute and 'TContext :> Context>(logger: ILogger) =
     do TASTCollecting.logger <- logger
 
     let registeredAnalyzers =
         ConcurrentDictionary<string, Client.RegisteredAnalyzer<'TContext> list>()
 
-    new() = Client(Abstractions.NullLogger.Instance, Set.empty)
+    new() = Client(Abstractions.NullLogger.Instance)
+
+    member x.LoadAnalyzers(dir: string) : int * int = x.LoadAnalyzers(dir, None)
 
-    member x.LoadAnalyzers(dir: string) : int * int =
+    member x.LoadAnalyzers(dir: string, excludeInclude: ExcludeInclude option) : int * int =
```

Comment:
Taking the filter argument here instead of in the constructor makes life easier for editor clients as they can hold onto a simple client instance and change the filter on the fly while reloading analyzers after a config change.

Reply:
Should this not be `?excludeInclude: ExcludeInclude`? Then it probably doesn't need to be a separate member.

Reply:
Thanks, that's nicer.